examples/python/easy_start_demo/README.md

```diff
@@ -0,0 +1,35 @@
+## easyDemo for using GRPC in Python
+###主要是介绍了在Python中使用GRPC时, 进行数据传输的四种方式。
+###This paper mainly introduces four ways of data transmission when GRPC is used in Python.
+
+- ####简单模式 (Simple)
+```text
+没啥好说的,跟调普通方法没差
+There's nothing to say. It's no different from the usual way.
+```
+- ####客户端流模式 (Request-streaming)
```

Comment:
Broadly speaking, within gRPC, we use the term "client streaming". Within Python specifically, we use the term "stream-unary". You may want to include both terms.